### ISSUE TITLE
Feature request: Long press for context menu

### ISSUE THREAD

**Issue by thanxx** [yuliskov/SmartYouTubeTV#424]
On some devices, I noticed the new feature in the official app: a long press to open the context menu
Can we have it as well?

![youtube](https://user-images.githubusercontent.com/21316174/72727474-aef8e580-3b93-11ea-919d-2c466803269a.png)

**Comment by onlytanmoy** [comment]
Thanks for this feature request. I would like to see this implemented as well.
Devs- Please provide this option to enable context menu with 'watch later'/'save to to playlist' etc. on long pressing a video...currently it starts playing the video straight away.

**Comment by ScottBeeson** [comment]
Yes, I would really like to have this!  I use it to tell Youtube to stop recommending me things my Son searched for :D

**Comment by onlytanmoy** [comment]
@thanxx @ScottBeeson Latest version of the app has this feature packed in now :) all thanks to @yuliskov 
Check out below issue for details
https://github.com/yuliskov/SmartYouTubeTV/issues/452

**Comment by onlytanmoy** [comment]
@yuliskov @fgl27 since today morning long hold for context menu is not working 😐 it's directly opening the video. Animated preview is working fine though. Please check.

**Comment by thanxx** [comment]
I can't use the feature as well in the new build, howeverer saw it in the changelog during the installation

**Comment by onlytanmoy** [comment]
@thanxx yeh it was working all fine until yesterday night. I spotted the issue today morning ☹️

**Comment by thanxx** [comment]
Can confirm it works in recent 'bugfix' release
Thank you guys

**Comment by thanxx** [comment]
However I had to enable this option in the settings, so it was off by default

**Comment by onlytanmoy** [comment]
Yep, this is working now after the latest update today 👍 Thanks guys! 😊

**Comment by onlytanmoy** [comment]
@thanxx please confirm one thing for me mate...when you search for something and then long press on any of the search results, do you just get Play option and not the save to watch later, save to playlist?
